### ISSUE TITLE
Feature Request: Add/Edit DHCP Leases

### ISSUE THREAD

**Issue by deefdragon** [marshallford/terraform-provider-pfsense#21]
I use Terraform to spin up my VMs. I can set a dedicated mac address on the VMs, and would love the ability to add that mac address to my PFSense DHCP Leases automatically.

**Comment by marshallford** [comment]
Yeah great call out, this is a feature I've been wanting to implement but haven't gotten around to it. Any specific behaviors/fields you are looking for? PfSense has a ton of "advanced options" available when creating a static mapping and it will be a pain to model all of them out of the gate.

**Comment by deefdragon** [comment]
I honestly only need the string fields in the main/first section when adding a static mapping. tho I do have a number of vans so the ability to set which vlan(network) is being used for a mapping is important to me as well.

**Comment by marshallford** [comment]
Just released `v0.9.1` which includes two new resources (`pfsense_dhcpdv4_staticmapping` and `pfsense_dhcpdv4_apply`) as well as one new data source (`pfsense_dhcpdv4_staticmappings`). Give them a whirl  and let me know if you bump into limitations, bugs, or poor documentation.

Couple other notes (mostly for myself 😆)
1. I'm considering if I should rename everything to `dhcpv4` instead of `dhcpdv4`.
2. I plan on adding some baseline validation to all of the resources here soon.
